src/Compilers/VisualBasic/Portable/Parser/Parser.vb

```diff
@@ -31,7 +31,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
         Private _possibleFirstStatementOnLine As PossibleFirstStatementKind = PossibleFirstStatementKind.Yes
         Private _recursionDepth As Integer
         Private _evaluatingConditionCompilationExpression As Boolean
-        Private ReadOnly _scanner As Scanner
+        Friend ReadOnly _scanner As Scanner
```

Comment:
>Friend ReadOnly _scanner As Scanner [](http://example.com/codeflow?start=8&length=35)

Do we need access to Scanner or to VisualBasicParseOptions? If latter, consider adding a property for options instead.